src/network/RTypeClient.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** RTypeClient
+*/
+
+#include <stdlib.h>
+#include "RTypeClient.hpp"
+
+RTypeClient::RTypeClient(std::string hostname, std::string port)
+: UDPConnection(hostname, port), gameEnd(false)
+{
+}
+
+void RTypeClient::engineInit()
+{
+    std::cout << "Init Game" << std::endl;
+    send("start");
+
+    uint16_t compNb = receiveUint16();
+    uint8_t compNameMaxSize = receiveUint8();
+
+    std::cout << "CompNb : " << (int)compNb << "." << std::endl;
+    std::cout << "CompNameMaxSize : " << (int)compNameMaxSize << "." << std::endl;
+
+    for (uint16_t i; i < compNb; i++) {
+        std::vector<uint8_t> compName = blockingReceive();
+        std::string strCompName = std::string(compName.begin() + 1, compName.end());
+        // uint16_t CompId = (static_cast<uint16_t>(compName[0]) << 8) | static_cast<uint16_t>(compName[1]);
```

Comment:
Is the debug needed in this function ? If not, can you remove it ?

Reply:
Do you mean the commented line ?